frontend/tests/playwright/fixtures/metrics-reporter.ts

```diff
@@ -112,9 +112,23 @@ class MetricsReporter implements Reporter {
       });
    }
 
+   // format the results into a file to read from stdin for the report-test-run script
+   createJsonResult() {
+      const resultFileName = `playwright-result-${this.commit}.txt`;
+      try {
+         const jsonString =
+            this.results.map((result) => JSON.stringify(result)).join('\n\n') +
+            '\n\n';
```

Comment:
I forget, why do we need to join on `\n\n`? If I remember right, we only need to do that at the end.

Reply:
According to https://github.com/iFixit/ifixit/blob/c9346d3c71d4fc09b13cff19e2ed7f188979f27c/Tests/Exec/report-test-run#L23, it said: `Note: you can send as many json objects as needed (separated by blank lines)`. 

This was in order to format the result file to look like this:

```
{"id":"abc", ....}

{"id":"abc", ....}

{"id":"abc", ....}

```

I tried running `Tests/Exec/report-test-run` without the blank lines in the test result file but it kept erroring out.